src/systems/ec/implementations/vanilla/packages/ec-component-form-select/ec-component-form-select.js

```diff
@@ -184,6 +180,28 @@ export class Select {
     this.search.classList.add('ecl-text-input', 'ecl-text-input--m');
     this.search.setAttribute('type', 'text');
     this.search.setAttribute('placeholder', this.textSearch || '');
+    this.search.addEventListener('keyup', e => {
+      const keyword = e.target.value;
+      if (keyword.length > 2) {
+        this.checkboxes.forEach(checkbox => {
+          if (
+            !checkbox
+              .getAttribute('data-select-multiple-value')
+              .includes(keyword)
+          ) {
+            checkbox.style.visibility = 'hidden';
```

Comment:
here we should toggle display instead of visibility (`display: none` or `display: flex`).
As the checkboxes are not used in the form itself, it can be done, and it will improve the design